bazel_tools/distribute_jar_cleanup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+SRC=$1
+OUT=$2
+NOTICE=$3
+LICENSE=$4
+LICENSE_EE=$5
+IS_EE=$6
+
+cp $SRC $OUT
+chmod +w $OUT
+zip -dq $OUT /LICENSE* /META-INF/LICENSE* /NOTICE* /META-INF/NOTICE*
```

Comment:
If those asterisks are supposed to reach `zip` (rather than be expanded by `bash`), should we put double-quotes around these?

Reply:
interesting, they do reach zip, but if the quotes are standard ill add them

Reply:
Ah gotcha, alright - i'll quote it :)